### DISCUSSION TITLE
Preservation Method v1 - CDE 7351590

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#50] [API Design Discussion]
caDSR Link: [7351590](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=7351590%20and%20ver_nr=1)

Definition: "The overall strategy for the collection of double stranded DNA fragments flanked by oligonucleotide sequence adapters to enable their analysis by high-throughput sequencing."

Issues with CDE:
| Permissible Value | VM Long Name | VM Description |
| --- | --- | --- |
| -80 Deg C | -80 Degrees Celsius | A quantity having a value less than zero.::1: A natural number greater than seventy-nine and less than eighty-one and the quantity that it denotes.::2: A unit of temperature of the temperature scale designed so that the freezing point of water is 0 degrees and the boiling point is 100 degrees at standard atmospheric pressure. The current official definition of the Celsius sets 0.01 C to be at the triple point of water and a degree to be 1/273.16 of the difference in temperature between the triple point of water and absolute zero. One degree Celsius represents the same temperature difference as one Kelvin. |
| FFPE | Formalin-Fixed Paraffin-Embedded | A colorless poisonous gas synthesized by the oxidation of methanol and used as an antiseptic, disinfectant, histologic fixative, and general-purpose chemical reagent for laboratory applications. Formaldehyde is readily soluble in water and is commonly distributed as a 37% solution in water; formalin, a 10% solution of formaldehyde in water, is used as a disinfectant and to preserve biological specimens. Environmentally, formaldehyde may be found in the atmosphere, smoke from fires, automobile exhaust and cigarette smoke. Small amounts are produced during normal metabolic processes in most organisms, including humans.: Tissue which is preserved in a fixative solution, usually formalin.: Tissue that is preserved and embedded in paraffin. |
| Frozen | Frozen Specimen | A specimen that has been subjected to and immobilized by severe cold. |
| LN2 | Liquid nitrogen | A substance in the fluid state of matter having no fixed shape but a fixed volume.: Nitrogen |

**FFPE** and **Frozen** refer to how the tissue or specimen is being preserved.   However, **-80 Deg C** and **LN2** only define the words being used.  

Also, adding a _null_ option ex.(**Unknown** or **Other**) would be useful.  The Federation API allows for data elements to return blank values when the unharmonized data cannot be harmonized to a permissible value in the CDE, so this likely not a priority for implementing this CDE.   

Potential Solutions:

- Subset this CDE to only allow for using the Permissible Values **FFPE** and **Frozen**.
- Update the CDE with definitions more fitting to the idea that -80C temperatures or liquid nitrogen is being used to preserve a sample biospecimen.

**Comment by jknable** [comment]
We recommend using CDE 8028962 (Biospecimen Preservation Procedure Type).  

The definitions for liquid nitrogen and -80 C in this CDE are the NCI Thesaurus term definition:
•	Liquid nitrogen - definition: An indication that a material has been stored in liquid nitrogen.
•	-80 degrees C - definition: A temperature of minus 80 Celsius.

**Comment by lydiabrogers** [comment]
Harmonization team meeting 2/16 - Recommend adopting [5432521](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=5432521%20and%20ver_nr=1.0) Tissue Sample Preservation Method Type. @jknable will this work?

**Reply by calkinsh** [reply]
For CHOP, the values in this CDE are better than in the original CDE, but because this is focused on tissue it does not have adequate terms to describe Blood preservation methods (such as EDTA). We do have blood/germline samples in our data so it would be helpful to be able to describe those. At a minimum, we'd need at least an "Other" value here to be able to describe those in some way.

**Comment by GeoffLyle** [comment]
Discussion has moved to #64 on whether we should adopt Preservation Method for CDE [5432521](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=5432521%20and%20ver_nr=1.0)
